config/nginx/default.conf

```diff
@@ -28,6 +28,27 @@ server {
 		fastcgi_pass phpfpm:9000;
 		fastcgi_index index.php;
 		fastcgi_param SCRIPT_FILENAME $document_root$fastcgi_script_name;
+
+
+        fastcgi_connect_timeout                 10;
+        fastcgi_send_timeout                    180;
+        fastcgi_read_timeout                    180;
+        fastcgi_buffer_size                     512k;
+        fastcgi_buffers                         4       256k;
+        fastcgi_busy_buffers_size               512k;
+        fastcgi_temp_file_write_size            512k;
+        fastcgi_intercept_errors                on;
+        fastcgi_keep_conn                       on;
+
+        # uncomment these for HTTPS usage
+        #fastcgi_param  HTTPS                   $https if_not_empty;
```

Comment:
I think we can just leave this particular line uncomment always and It should work fine on the backend